src/main/java/study/racinggame/domain/Cars.java

```diff
@@ -28,4 +32,25 @@ public static Cars copyCars(Cars cars) {
   private static Car copyCar(Car car) {
     return new Car(car.name(), car.distance());
   }
+
+  public void forwardCars() {
+    cars.forEach(car -> car.forward(new RacingGameStrategy()));
+  }
+
+  public List<Car> cars() {
+    return Collections.unmodifiableList(cars);
+  }
+
+  public List<String> carNamesAtLongestDistance() {
+    return cars().stream()
+            .filter(car -> car.isSameDistance(winnerCar()))
+            .map(Car::name)
+            .collect(Collectors.toList());
```

Comment:
```suggestion
    Car winner = winnerCar();
    return cars().stream()
            .filter(car -> car.isSameDistance(winner))
            .map(Car::name)
            .collect(Collectors.toList());
```

cars 수만큼 winnerCar()를 수행할 텐데요. 지역변수로 두는건 어떨까요?